azure-pipelines.yml

```diff
@@ -0,0 +1,19 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+- main
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+- script: echo Hello, world!
+  displayName: 'Run a one-line script'
+
+- script: |
+    echo Add other tasks to build, test, and deploy your project.
+    echo See https://aka.ms/yaml
+  displayName: 'Run a multi-line script'
```

Comment:
is it worth getting rid of this entirely until we do the corresponding work

Reply:
i added it as part of the initial task to setup the repo and run the azure pipeline using github repo. 

But indeed is  not doing anything .will get rid of